lms/views/api/grading.py

```diff
@@ -145,6 +151,7 @@ def record_canvas_speedgrader_submission(self):
         self.request.registry.notify(
             LTIEvent.from_request(request=self.request, type_=LTIEvent.Type.SUBMISSION)
         )
+        self.request.add_finished_callback(log_retries_callback)
```

Comment:
Only applying this and (err.retryable =  True) on this view.

If we merge this approach I'll create an issue to track generalizing this approach for other views.